pkg/lib/patternfly/patternfly-4-overrides.scss

```diff
@@ -342,6 +342,11 @@ svg {
     }
   }
 
+  // Not setting this breaks pixel tests
+  .screenreader-text {
+    right: -999px;
+  }
```

Comment:
That feels like a weird place. All the other attributes are set in pkg/shell/nav.scss, including `left: -999px`. Shouldn't that rather move there? This would keep all settings together, and this isn't a PatternFly class either.

Reply:
Indeed. I thought that all dark theme overrides are here, but it's my mistake. Will move.